addons/coupon/wizard/coupon_generate.py

```diff
@@ -5,7 +5,7 @@
 
 
 class SaleCouponGenerate(models.TransientModel):
```

Comment:
```suggestion
class CouponGenerate(models.TransientModel):
```

Reply:
@mreficent Thanks for the suggestions man. I'll take them into consideration.